cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -324,6 +325,27 @@ func stringList(s string) []string {
 	return spl
 }
 
+func checkInstanceConnectivity(instanceName string, public bool, client *proxy.Client) error {
+	// The connectivity test right now is just a basic dial to make sure it is reachable
+	// and nothing more. This will help rule out basic network connectivity problems, though
+	// such as firewalls and the like.
```

Comment:
nit: 
```suggestion
	// The connectivity test right now is just a basic dial to make sure it is reachable
	// and nothing more. This will help rule out basic network connectivity problems,
	// such as firewalls and the like.
```

Reply:
Done.